hydra-redis/redisgraph_demo1.py

```diff
@@ -0,0 +1,294 @@
+import redis
+from redisgraph import Graph, Node, Edge
+import urllib.request as ur
+import json
+import re
+from re import compile as regex
+from httplib2 import Http
+from hydrus.hydraspec import doc_maker
+
+
+redis_con = redis.Redis(host='localhost', port=6379)
+redis_graph = Graph("apidoc", redis_con)
+
+
+class _MemCache(dict):
+
+    def __nonzero__(self):
+        # even if empty, a _MemCache is True
+        return True
+
+    def set(self, key, value):
+        self[key] = value
+
+    def delete(self, key):
+        if key in self:
+            del self[key]
+
+
+DEFAULT_HTTP_CLIENT = Http(_MemCache())
+http = DEFAULT_HTTP_CLIENT
+APIDOC_RE = regex(
+    r'^<([^>]*)>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"$')
+
+
+def final_file(url):
+    """ open the given url and read and load the Json data.
+    """
+    response = ur.urlopen(url)
+    return json.loads(response.read().decode('utf-8'))
+
+
+def objects_property(objects_node, new_list, url_node):
+    """makes nodes for those properties which have properties in itself.
```

Comment:
Docstrings usually are in one line and start with a capital letter. The qoutes should also be in the same line for a single line docstring.
`"""This is an example of a doctstring."""`

Reply:
It would be better if you follow this format https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

Reply:
okay, I'll update it.